pkg/storage/chunk/predicate.go

```diff
@@ -0,0 +1,16 @@
+package chunk
+
+import (
+	"github.com/prometheus/prometheus/model/labels"
+
+	"github.com/grafana/loki/pkg/logql/syntax"
+)
+
+type Predicate struct {
+	Matchers []*labels.Matcher
```

Comment:
Matchers are for selecting the series, right?

Reply:
Right, these are the label selectors.